Description: Sign MokManager with sbsigntool instead of pesign
 Ubuntu infrastructure uses sbsigntool for all other EFI signing, so we use
 the same thing for signing MokManager with our ephemeral key.  This also
 avoids an additional build dependency on libnss3-tools.
Author: Steve Langasek <steve.langasek@canonical.com>
Forwarded: not-needed

Index: shim/Makefile
===================================================================
--- shim.orig/Makefile
+++ shim/Makefile
@@ -88,8 +88,8 @@
 		-j .debug_line -j .debug_str -j .debug_ranges \
 		--target=efi-app-$(ARCH) $^ $@.debug
 
-%.efi.signed: %.efi certdb/secmod.db
-	pesign -n certdb -i $< -c "shim" -s -o $@ -f
+%.efi.signed: %.efi shim.crt
+	sbsign --key shim.key --cert shim.crt $<
 
 clean:
 	$(MAKE) -C Cryptlib clean
